openedx/adg/lms/applications/helpers.py

```diff
@@ -784,3 +784,30 @@ def get_users_enrolled_in_prereqs(users_ids, update_bu_prereq=False):
         users_ids, prereq_course_groups
     )
     return users_enrolled_in_prereqs
+
+
+def is_course_in_prereqs(course, update_bu_prereq=False):
```

Comment:
Please change the name of the argument `update_bu_prereq`. We are not updating any flag here, just checking for presence of course in the pre_req or bu_prereq categoy. We can name the argument it to just "bu_prereq".